tensorflow/1.10/README.md

```diff
@@ -27,7 +27,7 @@ error-prone steps like task deployment, ClusterSpec configuration, task monitori
 1. install TensorFlow cluster with the following command from the DC/OS CLI from within `dcos-tensorflow-tools`:
```

Comment:
It seems like there are some formatting errors in this section (from lines 25 to 35).

- "Install Tensor" -> "Install TensorFlow"
- "1. install TensorFlow" -> "1. Install TensorFlow"
- there's an extra "x" at the end of line 35